pkg/transaction/transaction.go

```diff
@@ -107,10 +107,15 @@ type transactionService struct {
 	store   storage.StateStorer
 	chainID *big.Int
 	monitor Monitor
+
+	boostTipRange   []int
+	boostPriceRange []int
+	defaultBoost    bool
```

Comment:
Would it make sense to have new type (say `BoostConfig`) which would hold these tree variables? 

It would make code little bit more cleaner because:
- passing one value would be easier then passing three values
- it would group similar configs under singe type